src/main/kotlin/com/team4099/robot2022/config/Constants.kt

```diff
@@ -114,4 +114,14 @@ object Constants {
     const val THROTTLE_DEADBAND = 0.05
     const val TURN_DEADBAND = 0.05
   }
+
+  object ShooterConstants {
+    const val LEADER_MOTOR_ID = 0
+    const val FOLLOWER_MOTOR_ID = 1
+
+    enum class ShooterState(private val value: Double) {
```

Comment:
dont make it private and change name of "value"

Reply:
instead of making it power of the motor, it should instead be velocitys we want the shooter to run at